src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -423,7 +425,9 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
             PrecisionsRestriction::create<ngraph::opset1::GroupConvolution>({
                 {{0}, {ngraph::element::u8, ngraph::element::i8}},
                 {{1}, {ngraph::element::i8}}
-            })
+            }),
+            PrecisionsRestriction::create<ngraph::opset5::LSTMSequence>({}),
+            PrecisionsRestriction::create<ngraph::opset6::GRUSequence>({})
```

Comment:
Why do explicitly set PrecisionsRestriction for LSTM/GRU here? Isn't "lptPassConfig->disable<ngraph::pass::low_precision::RecurrentCellTransformation>();" enough here?

Reply:
@v-Golubev Please comment.
I think it is required for some piece of code logic.

Reply:
@dmitry-gorokhov these restrictions allow us to avoid some additional work starting from LPT Markup passes. E.g. thanks to that all FQ layers before Sequences don't be decomposed (whereas without these restrictions LPT has to decompose the corresponding FQ and fuse it back after common passes (with disabled RecurrentCellTransformation) part)